build/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# There's no way to build musl binaries with github
+# so we need to compile it inside a container
+# and manually push the binary to github
+
+VERSION=${VERSION:=0.18}
```

Comment:
We probably want to automate this step and the upload phase. For now, can we  pull and increment the version automatically, rather than hard-coding? Can use this to list recent versions: https://docs.github.com/en/rest/reference/repos#list-releases